index.html

```diff
@@ -154,29 +165,29 @@ <h2 id="project_tagline"></h2>
 
 <p><strong>One message:</strong></p>
 
-<pre><code>$ curl -H "Content-Type: application/json" -X POST -d '["test"]' http://localhost:8080/producer/msg?rk=test
+<pre><code>$ curl -H "Content-Type: application/json" -X POST -d '["{"data":{"outcome":{"conclusion":"TIMED_OUT","description":"Compilation timed out."},"persistentLogs":[{"name":"firstLog","uri":"http://myHost.com/firstLog"},{"name":"otherLog","uri":"isbn:0-486-27557-4"}]},"links":{"activityExecution":"aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee1","flowContext":"flowContext","causes":["cause1","cause2"]},"meta":{"domainId":"example.domain","id":"9cdd0f68-df85-44b0-88bd-fc4163ac90a0","type":"eiffelactivityfinished","version":"0.1.7","time":1478780245184,"tags":["tag1","tag2"],"source":{"host":"host","name":"name","uri":"http://java.sun.com/j2se/1.3/","serializer":{"groupId":"G","artifactId":"A","version":"V"}}}}"]' http://localhost:8080/producer/msg?ud=fem001&mp=eiffelsemantics
```

Comment:
only use "ud" in one example since that is a special case. In that example it should be clearly stated that the us is used.